client/src/component/qa/AddQuestion.jsx

```diff
@@ -45,57 +70,62 @@ const AddQuestion = ({ productId }) => {
-                ]
-              </h5>
-              <p>(1) Your Question *</p>
-              <textarea maxLength="1000" name="questionBody" placeholder="Why did you like the product or not?" required onChange={(e) => { handleAddQuestion(e); }} />
-              <p>(2) What is your nickname? *</p>
-              <input maxLength="60" placeholder="Example: jackson11!" name="nickname" required onChange={(e) => { handleAddQuestion(e); }} />
-              <p>For privacy reasons, do not use your full name or email address</p>
-              <p>(3) Your email *</p>
-              <input maxLength="60" placeholder="Example@example.com" name="email" required onChange={(e) => { handleAddQuestion(e); }} />
-              <p>For authentication reasons, you will not be emailed.</p>
-              <button type="submit" onClick={handleClose}>Submit</button>
-            </form>
-          </Modal.Body>
-        </Modal>
-      </Container>
+      <AddButton onClick={handleShow}> Add A Question </AddButton>
+      {show
+        ? (
+          <Container>
+            <Modal>
+              <form style={{ display: 'inline-block' }} onSubmit={handleSubmit}>
+                <h4 style={{ fontSize: '20px' }}>Ask Your Question</h4>
+                <h5 style={{ fontSize: '12px' }}>
+                  About the
+                  {' '}
+                  [
+                  {productName}
+                  ]
+                </h5>
+                {question.length <= 0 || name.length <= 0 || email.length <= 0 ? <p style={{ color: 'red', fontSize: '10px' }}>You must enter the followings:</p> : null}
+                <p style={{ fontSize: '15px' }}>(1) Your Question *</p>
+                <textarea style={{ width: '80%', height: '15%' }} maxLength="1000" name="questionBody" placeholder="Why did you like the product or not?" required onChange={(e) => { handleAddQuestion(e); }} />
+                <p style={{ fontSize: '15px' }}>(2) What is your nickname? *</p>
+                <input style={{ width: '80%', height: '8%' }} maxLength="60" placeholder="Example: jackson11!" name="nickname" required onChange={(e) => { handleAddQuestion(e); }} />
+                <p style={{ fontSize: '10px' }}>For privacy reasons, do not use your full name or email address</p>
+                <p style={{ fontSize: '15px' }}>(3) Your email *</p>
+                <input style={{ width: '80%', height: '8%' }} maxLength="60" placeholder="Example@example.com" name="email" required onChange={(e) => { handleAddQuestion(e); }} />
+                <p style={{ fontSize: '10px' }}>For authentication reasons, you will not be emailed.</p>
+                <button
+                  style={{ fontSize: '15px', cursor: 'pointer' }}
+                  type="submit"
+                >
+                  Submit
+
+                </button>
+                <button style={{ marginLeft: '50px', fontSize: '15px', cursor: 'pointer' }} type="button" onClick={handleClose}>Cancel</button>
```

Comment:
i think it's fine to merge these anyways, but if you have a lot of extra time on your hands, inline styling is not great and should be converted to styled components at least.
https://www.quora.com/Is-it-really-a-bad-practice-to-use-inline-styles-in-HTML

Reply:
Will do. thanks